Week07/lesson14/homework/responsive_articles_with_nav/css/styles.css

```diff
@@ -0,0 +1,229 @@
+/* Box Model Hack */
+* {
+ box-sizing: border-box;
+}
+
+/* Clearfix hack */
+.clearfix:before, .clearfix:after {
+  content: "";
+  display: table;
+}
+
+.clearfix:after{
+  clear: both;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+  font-size: 13px;
+  background-color: #F6F0E4;
+}
+
+/* Basic/Mobile Article Styling: */
+
+article {
+  padding: 20px;
+}
+
+h2 {
+  font-family: 'PT Serif', serif;
+  font-size: 2em;
+  margin: 20px 0;
+}
+
+p {
+  margin: 0;
+}
+
+img {
+  display: block;
+  width: 100%;
+}
+
+main {
+  margin: 0 auto;
+}
+
+header {
+  background-color: #7BAFA4;
+}
+
+nav {
+  width: 100%
+  padding-top: 60px;
+  padding-left: 10px;
+  padding-right: 10px;
+  clear: both;
+  overflow: hidden;
+}
+
+nav a:first-child{
+  border-top: 1px solid #F6F0E4;
+}
+
+nav a:last-child{
+  margin-bottom: 10px;
+}
+
+nav a {
+  border-bottom: 1px solid #F6F0E4;
+  text-decoration: none;
+  color: white;
+  display: block;
+  width: 100%;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  padding-left: 10px;
+  margin: 0 auto;
+}
+
+nav a:focus, nav a:hover {
+  background-color: #488A7C;
+}
+
+.hamburger {
+  width: 40px;
+  height: 40px;
+  float: right;
+  margin-top: 10px;
+  margin-right: 10px;
+  margin-bottom: 10px;
+}
+
+.hamburger-line {
+  margin-top: 7px;
+  margin-bottom: 7px;
+  height: 3px;
+  background-color: white;
+}
+
+.hidden {
+  max-height: 0px;
+  transition: all .5s ease-in-out;
+  -webkit-transition: all .5s ease-in-out;
+   -moz-transition: all .5s ease-in-out;
+   -ms-transition: all .5s ease-in-out;
+   -o-transition: all .5s ease-in-out;
```

Comment:
When you include prefixed properties, you include the prefixed ones first. That way, the minute the default/unprefixed one becomes supported by the browser standard, as it's further down the CSS, it'll override the prefixed/browser-specific one.

Also, as `transition` is nearly completely supported by modern browsers (http://caniuse.com/#feat=css-transitions) you don't really need the prefixed versions here